src/hotspot/share/gc/g1/g1SurvRateGroup.cpp

```diff
@@ -28,6 +28,8 @@
 #include "logging/log.hpp"
 #include "memory/allocation.hpp"
 
+#include "logging/logStream.hpp"
```

Comment:
Left-over debugging include?  I don't see any uses, but if I missed some then this needs to be put into proper
sort order.

Reply:
Leftover debug code. Removed. :(